src/main/java/bio/terra/common/FlightUtils.java

```diff
@@ -68,4 +68,28 @@ public static RetryRuleRandomBackoff getDefaultRandomBackoffRetryRule(final int
   public static RetryRuleExponentialBackoff getDefaultExponentialBackoffRetryRule() {
     return new RetryRuleExponentialBackoff(2, 30, 600);
   }
+
+  /**
+   * Given a {@link FlightContext} object, look to see if the there is a value in the input map and
+   * if not, read it from the working map
+   *
+   * @param context The FlightContext object to examine
+   * @param key The map key to attempt to read values from
+   * @param clazz Class used to deserialize the value from the map
+   * @param <T> The type of the expected value in the maps
+   * @return A typed value from the flight context with type T or null if no value is found
+   */
+  public static <T> T getContextValue(FlightContext context, String key, Class<T> clazz) {
+    // Typically, when this is used the bucket / storage account has been selected for this file.
+    // In the single file load case, the info is stored in the working map. In the bulk load case,
+    // the info is stored in the input parameters.
```

Comment:
This comment doesn't seem to apply this code.

Reply:
I left it in for context (since that's where it's used) but I think it's no longer relevant